library/open-liberty

```diff
@@ -35,6 +35,26 @@ Directory: releases/latest/full
 Architectures: amd64, ppc64le, s390x
 File: Dockerfile.ubuntu.adoptopenjdk11
 
+Tags: 21.0.0.4-kernel-slim-java8-openj9
+Directory: releases/21.0.0.4/kernel-slim
+Architectures: amd64, ppc64le, s390x
+File: Dockerfile.ubuntu.adoptopenjdk8
+
+Tags: 21.0.0.4-kernel-slim-java11-openj9
+Directory: releases/21.0.0.4/kernel-slim
+Architectures: amd64, ppc64le, s390x
+File: Dockerfile.ubuntu.adoptopenjdk11
+
+Tags: 21.0.0.4-full-java8-openj9
+Directory: releases/21.0.0.4/full
+Architectures: amd64, ppc64le, s390x
+File: Dockerfile.ubuntu.adoptopenjdk8
+
+Tags: 21.0.0.4-full-java11-openj9
+Directory: releases/21.0.0.4/full
+Architectures: amd64, ppc64le, s390x
+File: Dockerfile.ubuntu.adoptopenjdk11
+
 Tags: 21.0.0.3-kernel-slim-java8-openj9
```

Comment:
Shouldn't 21.0.0.3 be removed in both of these since it's superseded by 21.0.0.4?

Reply:
Our quarterly releases are meant to be longer term support releases. Our policy is to maintain N-2 releases actively, so we currently support 20.0.0.12, 21.0.0.3, and the current release of 21.0.0.4.